modules/hitlnext/src/backend/repository.ts

```diff
@@ -65,8 +66,9 @@ export default class Repository {
    * @param bp
    * @param timeouts Object to store agent session timeouts
    */
-  constructor(private bp: typeof sdk, private timeouts: object) {
+  constructor(bp: typeof sdk, private timeouts: object) {
```

Comment:
can you leave it as it was for 2 times the same reason .. code consistency
1- across the code base
2- on the same line you use the private keyword right there